src/Features/JsonPatch/src/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,2 @@
 #nullable enable
+~virtual Microsoft.AspNetCore.JsonPatch.Internal.PocoAdapter.TryConvertValue(object value, System.Type propertyType, out object convertedValue, Newtonsoft.Json.Serialization.IContractResolver contractResolver) -> bool
```

Comment:
Are there any other Adapters other than `PocoAdapter` that should be updated to take a `contractResolver`?

We should also keep `out object convertedValue` as the last parameter despite this causing a difference in overload parameter ordering.

> ✔️ DO place all out parameters following all of the by-value and ref parameters (excluding parameter arrays), even if it results in an inconsistency in parameter ordering between overloads (see [Member Overloading](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/member-overloading)).

https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/parameter-design